tide_event.module

```diff
@@ -123,38 +123,38 @@ function tide_event_webform_submission_presave($submission) {
     $event_details_data = [
       'type' => 'event_details',
       'field_paragraph_link' => [
-        'uri' => Html::escape($submission_data['website_url_for_booking']),
+        'uri' => Html::decodeEntities(Html::escape($submission_data['website_url_for_booking'])),
```

Comment:
@enzolutions Why do we need to escape/decode input data? Those utility functions from Html are for sanitising output.

Reply:
@sonnykt @GROwen requested to Sanitize the input of the user from FE, I am open to any suggestions

Reply:
@enzolutions @GROwen pls refer to https://www.drupal.org/docs/develop/security/why-does-drupal-filter-on-output

Reply:
I agree with you and that document, I already discuss that with @GROwen, so Guy is your call. we could remove those? and trust in Drupal Entity Saving validation for user Input?